### PR TITLE
Removed left join, unified entity integrity

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**

### CODE REVIEW

File: contents/concepts/01-Data-Model.rst [hunk 1]
```diff
@@ -9,31 +9,26 @@ What is a data model?
...
-Formalized almost a half-century ago in `A relational model of data for large shared data banks <https://dl.acm.org/citation.cfm?doid=362384.362685>`_, the relational data model provides the most rigorous approach to structured data storage and the most precise approach to data querying.
-The model is defined by the principles of data representation, domain constraints, uniqueness constraints, referential constraints, and declarative queries as summarized in Table 1.
-From a practical point of view, the relational model has many advantages over both hierarchical file systems and spreadsheets in terms of maintaining data integrity, building analysis workflows, and providing flexible access to interesting subsets of the data.
+Formalized almost a half-century ago in (`Codd, 1969 <https://dl.acm.org/citation.cfm?doid=362384.362685>`_), the relational data model provides the most rigorous approach to structured data storage and the most precise approach to data querying.
+The model is defined by the principles of *data representation*, *domain constraints*, *uniqueness constraints*, *referential constraints*, and *declarative queries* as summarized below.
```
Comment (austin-hilberg) [inline contents/concepts/01-Data-Model.rst:28]: Do we have any consistent guidelines for when to use *italics* and when to use **bold**? I have been using **bold** for introducing terminology and *italics* for emphasis of regular words, but I am not aware of any internal convention we may have.

File: contents/queries/05-Operators.rst [hunk 2]
```diff
@@ -28,22 +28,17 @@ operator                             notation                                  m
 
 Principles of relational algebra
 ---------------------------------
-DataJoint's relational algebra improves upon the classical relational algebra and upon other query languages to simplify and enhance the construction and interpretation of precise and efficient data queries.
+DataJoint'salgebra improves upon the classical relational algebra and upon other query languages to simplify and enhance the construction and interpretation of precise and efficient data queries.
```
Comment (austin-hilberg) [inline contents/queries/05-Operators.rst:31]: Missing a space here.

File: contents/concepts/02-Terminology.rst [hunk 3]
```diff
@@ -5,18 +5,18 @@
 Terminology
 ===========
 
-DataJoint introduces a principled data model, which is described in detail in `DataJoint: a simpler relational data model <https://arxiv.org/abs/1807.11104>`_.
+DataJoint introduces a principled data model, which is described in detail in (`Yatsenko et al., 2018 <https://arxiv.org/abs/1807.11104>`_).
```
Comment (austin-hilberg) [inline contents/concepts/02-Terminology.rst:8]: Syntactically, should we have parentheses if we are using the paper as a noun in the sentence?

File: contents/concepts/01-Data-Model.rst (hunk 1 above)
Comment (dimitri-yatsenko) [reply contents/concepts/01-Data-Model.rst:28]: yes, _italics_ for emphasis, **bold** for introducing new terms.

**Pull request merged**